### ISSUE TITLE
Color Contrast: Feed Tag

### ISSUE THREAD

**Issue by AutoSponge** [Hashnode/user-feedback#170]
Element has insufficient color contrast of 3.97 (foreground color: #808080, background color: #ffffff, font size: 10.5pt (14px), font weight: bold).

Element location: `.hn-card:nth-child(3) > .reply-info.with-tags.flex-wrap > p > a[href$="programming"]`
Element source: `<a href="/n/programming">#programming</a>`

**Expected behavior**
Expected contrast ratio of 4.5:1 or 3:1 for large type.

Suggested fix:
- Set font-size to `19px`
- 3.94:1 current ratio will pass, font is already bold.

**Comment by BolajiAyodeji** [comment]
Hi @AutoSponge,

Thanks for the amazing feedback. We are working on getting this fixed and we'll update you soon.
